base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
   font-family: "Roboto", sans-serif;
 }
+
+.gridLayout {
+  position: absolute;
+  min-width: 414px;
+  height: 100vh;
+  background-color: RGB(25, 30, 40);
+  display: flex;
+  flex-direction: column;
+}
+
+.navigationBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 400px;
+  height: 75px;
+  transition-duration: 5s;
+}
+
+.navigationBarToggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: rgba(200, 200, 200, 0.1);
+  width: 400px;
+  height: 75px;
+
+  transition-duration: 5s;
+}
+.mainArea {
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: transparent;
+
+  margin-left: 2%;
+}
+
+.menuIcon {
+  float: right;
```

Comment:
You don't need `float` here.
As you are using `flex`, the menu icon is already pushed to the right because all the menu items are before

Reply:
I used it iin my first attempt when i appended the items to the navigationBar.
Forgot to erase after following another approach. Thx

float right (removed)